templates/_environments.tpl

```diff
@@ -27,12 +27,16 @@ data:
 {{- end}}
 {{- end}}
 {{/*
-  coder.environments.configMapEnv adds an environment variable with the name 
-  of the config map holding additional user environment configuration.
+  coder.environments.configMapEnv contains two environment variables.
+  The first one contains the name of the config map.
+  The second one contains a JSON string representation of []v1.PodToleration objects.
+  This allows ce-manager to unmarshal them without going through the kube api-server.
 */}}
 {{- define "coder.environments.configMapEnv" }}
 {{- if .Values.serviceTolerations }}
 - name: ENVIRONMENT_CONFIG_MAP
```

Comment:
Is this used for anything anymore?

Reply:
It's not.

I checked ce-manager.

Looks like we can remove it from this chart and `kenv/environment.go`